### ISSUE TITLE
Create Navbar component

### ISSUE THREAD

**Issue by ImMattDavison** [immattdavison/NoMoreDomains-Website#15]
We need a responsive navbar component - this must be built using vanilla CSS.

**Comment by Burekslayer** [comment]
Hey, may I work on this one?

**Comment by ImMattDavison** [comment]
> Hey, may I work on this one? 

Hey @Burekslayer! I would love to assign you but currently trying to decide if React is the best option for this as I would like to include multiple pages. Bare with me.

**Comment by Burekslayer** [comment]
Ok, btw I didnt learn react yet (fairly new with node.js/express/mongoDB)

**Comment by anubhav2102** [comment]
Can i work on this?
I am familiar with React

**Comment by ImMattDavison** [comment]
> Can i work on this? I am familiar with React

We need to setup react-router first ideally - is this something you're familiar with?

**Comment by anubhav2102** [comment]
Yes, i am familiar with react-router-dom and other navigational tools of react.

**Comment by ImMattDavison** [comment]
> Yes, i am familiar with react-router-dom and other navigational tools of react.

May I recommend #16

**Comment by anubhav2102** [comment]
Ok Sure
